package/yast2-vm.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Jan 31 13:52:36 MST 2017 - carnold@suse.com
+
+- bsc#978225 - yast virtualization menu not updated after install
+  KVM and KVM tools
+- 3.1.31
```

Comment:
In `master` we already switched to `3.2.x` to avoid possible conflicts in the SP2, SP3 and CaaSP (a new product). Please start with `3.2.0` here.